Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM debian:stretch
 
 # Superset version
-ARG SUPERSET_VERSION=0.25.6
+ARG SUPERSET_VERSION=0.26.3
```

Comment:
I'd prefer to separate bumping Superset into a separate commit. Can you revert back to `0.25.6` for now?